backend/app/models/location_type.rb

```diff
@@ -2,7 +2,7 @@ class LocationType
   include EnumModel
 
   TYPES = %w[
-    state
+    country
     department
```

Comment:
do we then need to rename department to state?

Reply:
:D should I? I am fine with renaming `state` to `country` because it still makes sense for me, but in this case `department` still makes more sense to me then `state`. Also wiki and Santiago's data speak about these entities as departments. But if you wanna go for `state`, I will do it, no problem -- just please let me know your final word and I will tweak it eventually before merging this 🙏

Reply:
you're right, in the file it is called department, in the design state. Let's stick with department for now then.